src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs

```diff
@@ -63,9 +64,10 @@ await DataStore.CreateBuilder()
         // Assert
         Assert.That(report, Is.Not.Null);
         Assert.That(report.ReportData.EnvironmentInformation, Is.Not.Null, $"Environment information missing from the report");
-        Assert.That(report.ReportData.EnvironmentInformation.EnvironmentData, Is.Not.Null, $"Environment data missing from the report");
         Assert.Multiple(() =>
         {
+            Assert.That(report.ReportData.EnvironmentInformation.EnvironmentData, Is.Not.Null, $"Environment data missing from the report");
+
```

Comment:
Given that the additional newline was removed in quite a few other places what is the benefit of having it here?